src/fragments/lib-v1/start/getting-started/vanillajs/setup.mdx

```diff
@@ -0,0 +1,209 @@
+Create a new ‘plain’ JavaScript [ES2015](https://babeljs.io/docs/en/learn/) app with webpack. With the following commands, create the directory (`amplify-js-app`) and files for the app.
+
+```bash
+mkdir -p amplify-js-app/src && cd amplify-js-app
+touch index.html src/app.js webpack.config.js
+```
+
+Initialize npm and install dependencies and dev dependencies.
+
+```bash
+npm init
+npm install aws-amplify@next
+npm install webpack webpack-cli webpack-dev-server copy-webpack-plugin --save-dev
+```
+
+The app directory structure should be:
+
+```console
+amplify-js-app
+├── node_modules/
+├── src/
+│   └── app.js
+├── index.html
+├── package-lock.json
+├── package.json
+└── webpack.config.js
+```
+
+Add the following scripts to the `package.json` file:
+
+```json
+"scripts": {
+    "start": "webpack && webpack-dev-server --mode development",
+    "build": "webpack"
+}
+```
+
+Add the following to the `index.html` file:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Amplify</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <style>
+      html,
+      body {
+        font-family: 'Amazon Ember', 'Helvetica', 'sans-serif';
+        margin: 0;
+      }
+      a {
+        color: #ff9900;
+      }
+      h1 {
+        font-weight: 300;
+      }
+      hr {
+        height: 1px;
+        background: lightgray;
+        border: none;
+      }
+      .app {
+        width: 100%;
+      }
+      .app-header {
+        color: white;
+        text-align: center;
+        background: linear-gradient(30deg, #f90 55%, #ffc300);
+        width: 100%;
+        margin: 0 0 1em 0;
+        padding: 3em 0 3em 0;
+        box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.3);
+      }
+      .app-body {
+        width: 400px;
+        margin: 0 auto;
+        text-align: center;
+      }
+      .app-body button {
+        background-color: #ff9900;
+        font-size: 14px;
+        color: white;
+        text-transform: uppercase;
+        padding: 1em;
+        border: none;
+      }
+      .app-body button:hover {
+        opacity: 0.8;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="app">
+      <div class="app-header">
+        <h1>Welcome to Amplify</h1>
+      </div>
+      <div class="app-body">
+        <h1>Mutation Results</h1>
```

Comment:
We'll need to re-work this as DataStore won't be available in preview. Okay doing it in a follow-up PR